Makefile

```diff
@@ -290,11 +294,11 @@ build-and-deploy-ui:
 	&& if [ "$${DEPLOY_UI}" != "false" ]; then ${MAKEFILE_DIR}/devops/scripts/build_deploy_ui.sh; else echo "UI Deploy skipped as DEPLOY_UI is false"; fi \
 
 prepare-for-e2e:
```

Comment:
Not sure this target is needed anymore. I am probably the only person using it... I would probably just replace the reference to it here: https://github.com/microsoft/AzureTRE/blob/main/docs/tre-developers/end-to-end-tests.md?plain=1#L17  with concrete commands to run